README.md

```diff
@@ -147,30 +150,39 @@ acquire this dependency.
 2. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL of this repository,
    `https://github.com/microsoft/STL`.
 3. Open a terminal in the IDE with `` Ctrl + ` `` (by default) or press on "View" in the top bar, and then "Terminal".
-4. Invoke `git submodule update --init vcpkg` in the terminal.
-5. Invoke `.\vcpkg\bootstrap-vcpkg.bat` in the terminal.
-6. Invoke `.\vcpkg\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows` to install the boost-math dependency.
+4. In the terminal, invoke `git submodule update --init vcpkg`
+5. In the terminal, invoke `.\vcpkg\bootstrap-vcpkg.bat`
+6. In the terminal, invoke `.\vcpkg\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
 7. Choose the architecture you wish to build in the IDE, and build as you would any other project. All necessary CMake
    settings are set by `CMakeSettings.json`.
 
 # How To Build With A Native Tools Command Prompt
 
-These instructions assume you're targeting `x64-windows`; you can change this constant below to target other
-architectures.
-
 1. Install Visual Studio 2019 16.8 Preview 1 or later.
     * We recommend selecting "C++ CMake tools for Windows" in the VS Installer.
     This will ensure that you're using supported versions of CMake and Ninja.
     * Otherwise, install [CMake][] 3.17 or later, and [Ninja][] 1.8.2 or later.
-2. Open an "x64 Native Tools Command Prompt for VS 2019".
+2. Open a command prompt.
 3. Change directories to a location where you'd like a clone of this STL repository.
-4. Invoke `git clone https://github.com/microsoft/STL`
-5. Invoke `cd STL`
-6. Invoke `git submodule update --init vcpkg`
-7. Invoke `.\vcpkg\bootstrap-vcpkg.bat`
-8. Invoke `.\vcpkg\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows` to install the boost-math dependency.
-9. Invoke `cmake -G Ninja -S . -B {wherever you want binaries}` to configure the project. For example, `cmake -G Ninja -S . -B out\build\x64`
-10. Invoke `ninja -C {wherever you want binaries}` to build the project. For example, `ninja -C out\build\x64`
+4. `git clone https://github.com/microsoft/STL`
+5. `cd STL`
+6. `git submodule update --init vcpkg`
+7. `.\vcpkg\bootstrap-vcpkg.bat`
+8. `.\vcpkg\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+
+To build the x86 target:
+
+1. Open an "x86 Native Tools Command Prompt for VS 2019".
+2. Change directories to the previously cloned `STL` directory.
+3. `cmake -G Ninja -S . -B out\build\x86`
+4. `ninja -C out\build\x86`
+
+To build the x64 target:
+
+1. Open an "x64 Native Tools Command Prompt for VS 2019".

```

Comment:
This is sort of not totally true, the user wants an x64 targeting compiler right? Not explicitly the x64 native compiler. I guess it's ok to assume everyone's on x64 these days though so no changes requested.

Reply:
Note that this doesn't claim to the be the only possible way, just the recommended way.

(Also, the VS-installed clang-cl works in native command prompts only, not cross, so the difference is relevant for later testing.)